### PR TITLE
adding junit test output

### PR DESCRIPTION
**Pull request by leongold**

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by SchSeba** [comment]
ci add to whitelist

**Comment by leongold** [comment]
@phoracek @SchSeba pls review

File: automation/test.sh [hunk 1]
```diff
@@ -123,5 +123,7 @@ done
 
 kubectl version
 
+mkdir -p "$ARTIFACTS_PATH"
+ginko_params="--junit-output=$ARTIFACTS_PATH/tests.junit.xml"
```
[the remark this answers is not in the document]
Comment (leongold) [reply automation/test.sh]: Done

File: Gopkg.toml [hunk 2]
```diff
@@ -26,3 +26,7 @@
 [[constraint]]
   version = "8.0.0"
   name = "k8s.io/client-go"
+
+[[constraint]]
+  version = "0.1.2"
+  name = "kubevirt.io/qe-tools"
```
[the remark this answers is not in the document]
Comment (leongold) [reply Gopkg.toml]: Done

**Comment by phoracek** [comment]
LGTM, thanks @leongold @SchSeba

**Pull request merged**